blog/2015-03-31-Eclipse_Docker_Tooling.adoc

```diff
@@ -0,0 +1,90 @@
+= Upcoming Docker Tooling for Eclipse
+:page-layout: blog
+:page-author: xcoulon
+:page-tags: [jbosstools, docker, eclipse]
+
+We are currently busy working on Docker tooling for Eclipse and even though the
+code is not public yet (more details on that at the end of this blog entry), we thought it
+would be interesting to share our progress on the project.
+
+The Docker tooling is aimed at providing in Eclipse the same level features as
+users have with the command-line interface.
+We rely on the https://github.com/spotify/docker-client[open source Docker client
+developped by Spotify] to handle the low-level communication with the Docker daemons
+and we provide views and wizards to setup connections and manage images
+and containers.
+
+== Docker Explorer
+
+The *Docker Explorer* is a tree view that handles multiple connections and provides users
+with quick overview of the images and containers.
+
+image::images/docker_tooling_explorer_view.png[Docker Explorer view]
+
+Built-in filters can show/hide intermediate and 'dangling' images as well as
+stopped containers.
+
+A wizard lets the user establish a new connection. This wizard can detect default
```

Comment:
How about putting this before the explorer so the flow is in what sequence users will see it.